### DISCUSSION TITLE
Ignore templates in HTML attributes

### DISCUSSION THREAD

**Discussion by Anthony-Gaudino** [parcel-bundler/parcel#7596] [Q&A]
I would like to have HTML files with attributes, for example:
```html
<img src="${logo}"/>
```

I'm importing the HTML file into JS and then replacing the property.
The problem I'm facing is that Parcel detects this as an error because it tries to find a `${logo}` file which doesn't exist.

Is it possible to skip properties that have JS template like strings inside?

**Comment by devongovett** [comment]
You probably need to pre-process this with your templating language before passing it to Parcel, or use a transformer plugin to do that.
